src/components/Camera/PhotoPreview.js

```diff
@@ -62,7 +70,7 @@ const PhotoPreview = ( { photoUris, setPhotoUris }: Props ): Node => {
         <Modal
           visible={mediaViewerVisible}
           onDismiss={hideModal}
-          contentContainerStyle={viewStyles.container}
+          // contentContainerStyle={viewStyles.container}
```

Comment:
Please delete this comment and maybe `viewStyles.container` unless we're actually going to use it.